README.md

```diff
@@ -532,7 +532,17 @@ RANDOM_COLOR
 
 // Render every individual mesh in the scene with a random color.
 RANDOM_NODE_COLOR
+
+// Sets a custom color using the customDebugColor call back. 
+CUSTOM_COLOR_MODE
+```
+### .customDebugColor
+
 ```
+customDebugColor: (tile: Tile, child: Object) => void
+```
+
+The callback used if `debugColor` is set to `CUSTOM_COLOR_MODE`. Value default to `null` and must be explicitly set
```

Comment:
Can we call the field `customColorCallback` so it's more clear that it's a callback to be set. And thanks for updating the documentation!